custom-styles/generic/filters.ts

```diff
@@ -0,0 +1,34 @@
+import { createStyles } from "@material-ui/core";
+import theme from "../../theme";
+
+export default createStyles({
+
+  root: {
+
+  },
+
+  list: {
+
+  },
+
+  container: {
+
+  },
```

Comment:
Change all empty objects in every style file to single line, it shows more clearly, what overrides are actually used.

```suggestion
  root: { },

  list: { },

  container: { },
```